src/libraries/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj

```diff
@@ -1,8 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);XMLSERIALIZERGENERATORTESTS</DefineConstants>
+    <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
     <SkipTestsOnPlatform Condition="'$(TargetsMobile)' == 'true' or '$(TargetOS)' == 'FreeBSD' or '$(TargetArchitecture)' == 'arm' or '$(TargetArchitecture)' == 'arm64' or '$(TargetArchitecture)' == 'armel' or '$(TargetArchitecture)' == 'wasm'">true</SkipTestsOnPlatform>
+    <UseLocalTargetingRuntimePack>false</UseLocalTargetingRuntimePack>
```

Comment:
This deserves a comment.  Let's mention that this test assembly needs to run on the tool shared framework so it should reference the latest references.  We build against tool shared framework's ref pack, run the generator on the tool shared framework, then execute the test on the test shared framework.